plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeLegacyCreateTableWithExistingLocation.java

```diff
@@ -31,7 +30,6 @@
 import static io.trino.plugin.hive.metastore.file.FileHiveMetastore.createTestingFileHiveMetastore;
 import static io.trino.testing.TestingNames.randomNameSuffix;
 import static java.lang.String.format;
-import static org.assertj.core.api.Assertions.assertThat;
 
 public class TestDeltaLakeLegacyCreateTableWithExistingLocation
```

Comment:
I think we can remove this class and move testLegacyCreateTable to BaseDeltaLakeMinioConnectorTest with a new name , e.g. testCreateTableWithExistingLocation.